plutus-core/plutus-core/src/PlutusCore/Pretty/Readable.hs

```diff
@@ -272,3 +272,11 @@ iterInterAppPrettyM fun args =
     let ppArg (Left ty)    = prettyArg $ inBraces ty
         ppArg (Right term) = prettyArg term
     in prettyFun fun :| map ppArg args
+
+-- | Pretty-print something with the @PrettyConfigReadable@ config.
+prettyReadable :: (PrettyReadable a) => a -> Doc ann
+prettyReadable = prettyBy (botPrettyConfigReadable prettyConfigName def)
+
+-- | Pretty-print something with the @PrettyConfigReadableSimple@ config.
+prettyReadableSimple :: (PrettyReadable a) => a -> Doc ann
+prettyReadableSimple = prettyBy (botPrettyConfigReadable prettyConfigNameSimple def)
```

Comment:
I wonder how this wasn't done ages ago... I suppose because we used `prettyPlcReadable` instead, but in any case it's very sensible to have these functions here. Thanks!

Reply:
Right, we still have `prettyPlcReadable` - it should be no longer needed